Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -1866,6 +1887,9 @@ internal int GroupableColumnsCount
     internal bool IsFixedHeader
         => Virtualize || FixedHeader;
 

```

Comment:
Add an XML comment explaining the property.

Reply:
Can you clarify where you put this?
I see an unrelated feature?

![image](https://github.com/Megabit/Blazorise/assets/22283161/5ec751f7-29bd-4841-924a-0571bda683d1)

I am assuming I might have missed the new parameter? I'll take a look when I start writing the docs.

Reply:
No, just write the comment in the code. Not in the docs :)

Reply:
Yes, I understood, already done.